src/main/kotlin/racingcar/domain/RacingGame.kt

```diff
@@ -0,0 +1,23 @@
+package racingcar.domain
+
+private const val MIN_CARS_SIZE = 2
+
+class RacingGame(
+    private var cars: List<Car>,
+    private val tryNumber: TryNumber,
+) {
+    init {
+        require(cars.size >= MIN_CARS_SIZE)
+    }
+
+    fun play(movementStrategy: () -> Int): List<List<Car>> {
+        val result = mutableListOf<List<Car>>()
+        while (!tryNumber.isComplete()) {
```

Comment:
`isComplete()`를 `!`를 이용할 거라면 처음부터 `isNotComplete()`로 하는게 코드를 읽기 더 좋지 않을까요?
```suggestion
        while (tryNumber.isNotComplete()) {
```

Reply:
확실히 `!`를 붙인 부정문은 직관적으로 바로 이해가 안되는 것 같네요. 😃

Reply:
해당 코멘트에서는 조금 고민되는 부분이 있습니다! 

현재 `TryNumber#isComplete` 함수는 외부(=RacingGame)에서도 사용되고 있지만 내부(=consume함수)에서도 사용되고 있는데, 이 둘은 `게임이 진행중이 상태`, `게임이 끝난 상태` (반대의 상황)에서 모두 쓰이고 있어서 결국 `isComplete()`로 명칭을 짓던지 `isNotComplete()`로 명칭을 짓던지 결국 `!` 부정문의 사용을 피하기가 힘들 것 같습니다. 

그렇다고 별개의 메서드로 쪼개는 것은 과하다고 생각하는데 지금의 경우에는 따로 메서드를 분리하거나 `!`를 사용하는 방법밖에 없다고 생각하는데 저는 후자가 낫다고 판단하여 `!`를 사용하는 지금의 형태로 유지했습니다. 
(개인적으로 isPlaying이 조금 더 의미있다고 생각되서 네이밍 변경하였습니다!)

Reply:
저같은 경우에 그러면 두개의 메서드(`isComplete`, `isNotComplete`)를 전부 만드는 편입니다.
단순하게 단일책임의원칙으로 바라본다면 개별 메서드를 만드는 것이 옳다고 생각합니다!
다만 두 가지가 완벽하게 대칭점에 있고 공개 메서드라는점에서 `isComplete`를 구현했다면 `isNotComplete`는 `!isComplete`로 구현할 것같네요!